sources/logdog/src/log_request.rs

```diff
@@ -9,9 +9,7 @@
 //! these provide the list of log requests that `logdog` will run.
 
 use crate::error::{self, Result};
-use datastore::deserialization::from_map;
-use datastore::serialization::to_pairs;
-use glob::{glob, Pattern};
+use glob::glob;
```

Comment:
We can drop the `glob` dependency it looks like?

Reply:
I wish but it is used in another method i used udeps to validate

Reply:
its used by handle_glob_request